app/client/rpc/server.go

```diff
@@ -0,0 +1,37 @@
+package rpc
+
+import (
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/pokt-network/pocket/app"
+	"github.com/pokt-network/pocket/shared"
+)
+
+var APIVersion = app.AppVersion
+
+type rpcServer struct {
+	node *shared.Node
+}
+
+func NewRPCServer(pocketNode *shared.Node) *rpcServer {
+	return &rpcServer{
+		node: pocketNode,
+	}
+}
+
+func (s *rpcServer) StartRPC(port string, timeout uint64) {
+	log.Printf("Starting RPC on port %s...\n", port)
+
+	routes := s.GetRoutes()
+
+	srv := &http.Server{
+		ReadTimeout:       30 * time.Second,
```

Comment:
I think these are V0 default values XD

Let's not use any of these timeout values for now as they are specific to the performance issues of the IAVL tree

Reply:
Should we make it part of the config?